components/prize_pool/commons/prize_pool.lua

```diff
@@ -332,7 +335,14 @@ PrizePool.additionalData = {
 	LASTVSSCORE = {
 		field = 'lastvsscore',
 		parse = function (placement, input, context)
-			local scores = Table.mapValues(mw.text.split(input, '-'), tonumber)
+			local forceValidScore = function(score)
+				if Table.includes(SPECIAL_SCORES, score:upper()) then
+					return score:upper()
+				end
+				return tonumber(score)
+			end
+
+			local scores = Table.mapValues(Table.mapValues(mw.text.split(input, '-'), mw.text.trim), forceValidScore)
```

Comment:
using `'-'` in the split will still not allow negative scores
imo either make it `lastscore` and `lastvsscore` again or enable wikis to decide on their delimiter (with `-` as default)

Reply:
Yeah, but that's  #1541 :)